src/client/components/FilterCollapse.tsx

```diff
@@ -94,16 +94,31 @@ const FilterCollapse: React.FC<FilterCollapseProps> = ({
               }
             }}
           />
-          <Button color="primary" onClick={() => setFilterInput(searchFilterInput || '')}>
-            <span className="px-4 whitespace-nowrap">{buttonLabel}</span>
-          </Button>
+          <ResponsiveDiv md>
```

Comment:
Oddly grouping the buttons by <ResponsiveDiv> for desktop then mobile made the sizing all wrong, thus leaving as all Apply buttons then all Reset buttons